pegasus/sites.v3/code.org/public/css/hoc-banner.scss

```diff
@@ -1,5 +1,14 @@
 @import 'color.scss', 'font.scss', 'breakpoints.scss';
 
+#homepage #hero {
+  overflow: hidden;
+}
+
+#pageheader-wrapper #pageheader {
+  position: relative;
+  z-index: 10;
```

Comment:
Added this to fix the overlap issue preventing clicks in the top nav.

Reply:
out of curiousity, how did you come up with 10 here?